lib/llvm/import-llvm

```diff
@@ -42,13 +45,18 @@ Support_imports = [
 Support_imports += [
     'COFF', 'ConvertUTF', 'ConvertUTFWrapper', 'Debug', 'FileSystem',
     'FileUtilities', 'Host', 'Locale', 'Path', 'Process', 'Program', 'SMLoc',
-    'SourceMgr', 'Signals', 'Unicode', 'UnicodeCharRanges']
+    'SourceMgr', 'Signals', 'Unicode', 'UnicodeCharRanges', 'CommandLine',
+    'MachO', 'Error', 'StringSaver', 'thread', 'TargetParser',
+    'ARMBuildAttributes', 'ARMTargetParser', 'AArch64TargetParser']
```

Comment:
Hrm... I'm not sure what to do here because I am unlikely to have time in the near term to do the surgery myself, but I do know how to slice and dice the LLVM code.

BTW... I hang out on SwiftPM's Slack if you ever feel the need to chat in a more live fashion: https://swift-package-manager.herokuapp.com

If you want to push this forward, if you could find a way to get rid of the need for Triple etc via an extra diff on top of the changes which #if 0's out the problematic regions, that will at least make it more clear what we actually need.